packages/babel-plugin-transform-svg-component/src/util.js

```diff
@@ -171,6 +171,9 @@ export const getInterface = ({ types: t }, opts) => {
 }
 
 export const getImport = ({ types: t }, opts) => {
+  if (opts.useInfernoJsMode) {
+    return '';
+  }
```

Comment:
This line is covered by the new unit test written.
For example 
● webpack loader › should convert file for Inferno with props